### DISCUSSION TITLE
[4.0] Check in/out methods on users required?

### DISCUSSION THREAD

**Discussion by Flowman** [joomla/joomla-cms#32522] [General]
The two methods could be cleaned out from the code as the user tables does not have an checked_out column

https://github.com/joomla/joomla-cms/blob/60f2f0d793faf86ccda52ee2569fbebaf12277cc/components/com_users/src/Model/ProfileModel.php#L72

https://github.com/joomla/joomla-cms/blob/60f2f0d793faf86ccda52ee2569fbebaf12277cc/components/com_users/src/Model/ProfileModel.php#L103

**Comment by joomdonation** [comment]
That makes sense. Please note that you will also need to remove the code which calling the method from controller https://github.com/joomla/joomla-cms/blob/4.0-dev/components/com_users/src/Controller/ProfileController.php, too

Could you please make a PR for this change?

**Comment by joomdonation** [comment]
OK. So I went ahead to make PR https://github.com/joomla/joomla-cms/pull/32590/ to remove those methods
